overview.md

```diff
@@ -22,19 +22,19 @@ Open Source Discovery: Rapid scanning and identification of open source librarie
 
 Detect for Azure DevOps is architected to integrate seamlessly with build and release pipelines for Black Duck
 
-![extension](images/new-extension.png)
+![extension](images/bd-extension.png)
 
 ## Documentation ##
 
-Instructions and examples are available on our [Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Synopsys+Detect+for+Azure+DevOps)
+Instructions and examples are available on our [Public Confluence](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Black+Duck+Detect+for+Azure+DevOps)
 
-For information on the full capabilities of Detect visit [Detect Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect)
+For information on the full capabilities of Detect visit [Detect Public Confluence](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Black+Duck+Detect)
```

Comment:
Same goes for this link as well. Please put this link here: https://documentation.blackduck.com/bundle/detect/page/introduction.html